src/python/pants/bin/pants_exe.py

```diff
@@ -25,10 +26,10 @@ def test_env():
 
 
 def main():
-    start_time = time.time()
-
     with maybe_profiled(os.environ.get("PANTSC_PROFILE")):
+        start_time = time.time()
         try:
-            PantsRunner(start_time=start_time).run()
+            runner = PantsRunner(args=sys.argv, env=os.environ,)
```

Comment:
This bad trailing comma will now fail with Flake8 (merged a PR to do that a couple hours ago). You'll need to manually remove it.